modules/illumina.nf

```diff
@@ -99,6 +129,7 @@ process trimPrimerSequences {
         samtools index ${sampleName}.mapped.bam
         ${ivarCmd} -i ${sampleName}.mapped.bam -b ${bedfile} -m ${params.illuminaKeepLen} -q ${params.illuminaQualThreshold} -p ivar.out
         samtools sort -o ${sampleName}.mapped.primertrimmed.sorted.bam ivar.out.bam
+        samtools index ${sampleName}.mapped.primertrimmed.sorted.bam
```

Comment:
Is indexing this bam needed for anything? I don't think you should change trimPrimerSequences

Reply:
Thanks for review, I will revert the trimPrimerSequences step

Reply:
Need input cram which passes qc to test bamToCram generation